Jenkinsfile

```diff
@@ -82,7 +82,7 @@ pipeline {
                                                 credentials('brave-builds-github-token-for-pr-builder')
                                                 github('brave/devops', 'https')
                                             }
-                                            branch('${params.DEVOPS_BRANCH}')
+                                            branch('mschfh-brave-service-key-v2-jenkins')
```

Comment:
Just leaving a not here that this change needs to be undone before the PR gets merged.

Reply:
I will revert this commit after https://github.com/brave/devops/pull/11148 is merged on Monday